metadata_service/config.py

```diff
@@ -1,10 +1,22 @@
 import os
 
-
+# TODO: Remove the NEO4J keys and use the PROXY keys instead for NEO4J also
 NEO4J_ENDPOINT_KEY = 'NEO4J_ENDPOINT'
 NEO4J_USER = 'NEO4J_USER'
 NEO4J_PASSWORD = 'NEO4J_PASSWORD'
 
+# PROXY configuration keys
+PROXY_HOST = 'PROXY_HOST'
+PROXY_PORT = 'PROXY_PORT'
+PROXY_USER = 'PROXY_USER'
+PROXY_PASSWORD = 'PROXY_PASSWORD'
+PROXY_CLIENT = 'PROXY_CLIENT'
+
+
+PROXY_CLIENTS = {
```

Comment:
do you want to add a todo to update this part once atlas client is implemented?

Reply:
I already have that task in my JIRA ;) 
The only thing now I'm considering is to remove the NEO4J variables now, as the generic config variables can now be used instead.